content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -90,45 +92,45 @@ request rate doesn't affect the memory consumption.
 
 ### Latency
 
-Since Istio injects a sidecar proxy on the data path, latency is an important
+Since Istio injects a sidecar proxy or ztunnel proxy on the data path, latency is an important
 consideration.
 Every feature Istio adds also adds to the path length inside the proxy and potentially affects latency.
 
 The Envoy proxy collects raw telemetry data after a response is sent to the
```

Comment:
should we add ztunnel here as well?

Reply:
I'm not sure what you mean

Reply:
line 95 starts with sidecar or ztunnel, and line 99 only covers Envoy proxy. Should it also cover ztunnel as well in the explanation.

Reply:
I see